framework/src/meshgenerators/ParsedGenerateSideset.C

```diff
@@ -50,6 +50,10 @@ ParsedGenerateSideset::validParams()
       "A set of neighboring subdomain ids. A face is only added if the subdomain id of the "
       "neighbor is in this set",
       "included_neighbor_ids is deprecated, use included_neighbors with names or ids");
+  params.addParam<bool>(
+      "include_only_external_sides",
```

Comment:
It would be better for consistency to name the parameter the same way as the other object I linked to in the issue.

Reply:
it looks better here?
all the parameters are called include_xx, include_yy ?